at.nis.one_click_backup.yml

```diff
@@ -0,0 +1,30 @@
+{
+    "app-id": "at.nis.one_click_backup",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "6.6",
+    "sdk": "org.kde.Sdk",
+    "command": "NIS_One-Click-Backup_Qt",
+    "separate-locales": false,
+    "finish-args": [
+        "--share=ipc",
+        "--socket=wayland",
```

Comment:
so what's a plan when the system doesn't support wayland?

Reply:
I originally had "--socket=X11" too, but than it complained that this is redundant...

Reply:
I will add "--socket=fallback-x11" if that is okay?